Process and Policy Pages for Submitting Data (#1627)
agrc/gis.utah.gov: sharing/open-sgid/index.md

categories: []
---

> If you would like to share data with the [Open SGID]({% link sgid/index.html %}#open-sgid), we will work with you to get a copy of the data. We'll then run checks and reviews on the data to ensure they are clean before being loaded, and we also need to identify when and how we'll incorporate any future updates to the data.

Reviewer: ```suggestion
If you would like to share data with the [Open SGID]({% link sgid/index.html %}#open-sgid), we will work with you to get a copy of the data and identify when and how we will incorporate future updates to the data. We will also review the data to ensure they are free of common issues before being loaded.
```

We will review your submission against the qualifications and reach out with any questions. You can track this in the `AGRC's Process` section of the issue.

We run [Sweeper](https://github.com/agrc/sweeper) on all Open SGID datasets to fix empty or duplicate geometries/rows and to double check the metadata. If you'd like, we'll share Sweeper's results with you. Fixing it on your end will help streamline any future data updates, but this isn't necessary to submit the data for the Open SGID. We'll still happily add it to the Open SGID after fixing any issues on our local copy with Sweeper.

### Step 6: 📤 Publish (You and Us)
{: .text-left}

Once we've worked together to resolve any questions or problems, we'll add it to the Open SGID. 

> If you sent the data to us as a shpefile or GDB, or as a private or temporary feature service, we will also upload your dataset to our own ArcGIS Online organization as a hosted feature service and share it publicly through Open Data. As part of this process, we will add a thumbnail that includes both the AGRC logo and the relevant SGID category icon to help people searching ArcGIS Online identify SGID data:

Reviewer: ```suggestion
If you sent the data to us as a file or as a private or temporary feature service, we will also upload your dataset to our own ArcGIS Online organization as a hosted feature service and share it publicly through Open Data. As part of this process, we will add a thumbnail that includes both the AGRC logo and the relevant SGID category icon to help people searching ArcGIS Online identify SGID data:
```
